mmv1/products/documentaiwarehouse/Location.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Location'
+base_url: 'projects/{{project}}/locations/us:initialize'
+self_link: '{{name}}'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/document-warehouse/docs/overview'
+  api: 'https://cloud.google.com/document-warehouse/docs/reference/rest/v1/projects.locations'
+description: |
+  A location is used to initialize a project.
+skip_delete: true
+skip_sweeper: true
+# import_format:
+#   [
+#     'projects/{{project}}/locations/{{location}}',
+#     '{{location}}'
+#   ]
+# timeouts: !ruby/object:Api::Timeouts
+#   insert_minutes: 20
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "document_ai_warehouse_location"
+    primary_resource_id: "example"
+# custom_code: !ruby/object:Provider::Terraform::CustomCode
+#   custom_create: 'templates/terraform/custom_create/document_ai_warehouse_location.go'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    description: |
+      The location of the resource.
```

Comment:
It is better to give more information for the description of this field. It is OK to just copy the description from API doc.

Reply:
The API doc doesn't have any description, but I've added some more context

Reply:
Found some description from api doc
https://cloud.google.com/document-warehouse/docs/reference/rest/v1/projects.locations/initialize

```
The location to be initialized Format: projects/{projectNumber}/locations/{location}. It takes the form projects/{project}/locations/{location}.
```

Reply:
Sorry, you are right. There is not information for the field `location` in the API doc. Do you know what values are allowed for this field besides `us`?

Reply:
Other than `us`, `eu` is allowed